### ISSUE TITLE
Good Idea, Buggy implementation

### ISSUE THREAD

**Issue by Iulian94** [mrjamiebowman/Hangfire.JobManagement#1]
Please fix bugs to be usage completely as Hangfire is a known scheduling tool which is used with MassTransit

**Comment by mrjamiebowman** [comment]
You're not wrong. It is buggy but it would help if you specifically listed what you want fixed. This is an open-source community-driven project that I forked. I'm happy to help but saying it's buggy doesn't provide me any specifics on what needs to be fixed. Happy to help but please list actual things that specifically need fixing with reproducible steps. Or provide a GitHub sample of what's not working. Thank you!

**Comment by Iulian94** [comment]
The basic functionalities, crud interface. I tried it and basically can't do much. 
This has not changed much from a few years back when I had a project in which we actually tried customizing and making it work but only after months managed to make it work.
